### DISCUSSION TITLE
160

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#780] [Comments]
http://jojoldu.tistory.com/160

**Comment by jojoldu** [comment]
작성자: seungdols

$1로 값까지 복사시키는 방법은 신기하네요. 확실히 툴을 잘 다루는 것도 능력이네요!!^^
감사합니다.

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

승돌님 댓글 감사합니다! 
쓰면쓸수록 인텔리J 가 좋다는 생각이 들어서 기능을 찾아보는것 같습니다^^

**Comment by jojoldu** [comment]
작성자: inbonk

그루핑을 이렇게도 쓸 수 있었군요~
꿀팀 감솹니다~ ^^

**Comment by ors-woon** [comment]
신기한 기능이네요 
좋은글 잘보고 갑니다 ~~ :D
